handlers/root.go

```diff
@@ -17,14 +23,18 @@ func getStatus(w http.ResponseWriter, r *http.Request) {
 
 func GetToken(w http.ResponseWriter, r *http.Request) {
 	u, _, ok := r.BasicAuth()
+
 	if !ok {
 		RespondWithJSONIndent(w, http.StatusInternalServerError, u)
 		return
 	}
+
 	token, error := config.GenerateToken(u)
```

Comment:
"error" is a keyword in Go - I believe this should be named "err" instead - that's our convention and I don't think we want to be naming variables as the same as a Go keyword.

Reply:
Ok, this will be updated.